<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorEditForm</class>
 <widget class="QWidget" name="SensorEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="GroupBoxEx" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>70</number>
      </property>
      <item row="5" column="5">
       <widget class="LineEditEx" name="lineEditPort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="LineEditEx" name="lineEditValueMin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="LabelEx" name="label">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>传感器名称：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="LineEditEx" name="lineEditWaringUp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="LabelEx" name="label_5">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>颜色类型：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="LabelEx" name="label_3">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>传感器类型：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="LabelEx" name="label_11">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>报警上限：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="LabelEx" name="label_9">
        <property name="text">
         <string>最大值：</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="PushButtonEx" name="pushButtonOk">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="LineEditEx" name="lineEditValueMax">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="LabelEx" name="label_10">
        <property name="text">
         <string>报警下限：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="CheckBoxEx" name="checkBoxEnable">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>启用</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="LineEditEx" name="lineEditWaringDn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="LabelEx" name="label_2">
        <property name="text">
         <string>传感器编号：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="LabelEx" name="label_6">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>网关IP：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="LineEditEx" name="lineEditSensorNO">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="LabelEx" name="label_8">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>最小值：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="LineEditEx" name="lineEditIP">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="LineEditEx" name="lineEditSensorName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <widget class="PushButtonEx" name="pushButtonCancel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="LabelEx" name="label_7">
        <property name="text">
         <string>端口：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="LabelEx" name="label_4">
        <property name="text">
         <string>区域类型：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="LineEditEx" name="lineEditSensorType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="LineEditEx" name="lineEditColorType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="5">
       <widget class="LineEditEx" name="lineEditAreaType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupBoxEx</class>
   <extends>QGroupBox</extends>
   <header>ctrls/GroupBoxEx.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PushButtonEx</class>
   <extends>QPushButton</extends>
   <header>ctrls/PushButtonEx.h</header>
  </customwidget>
  <customwidget>
   <class>LineEditEx</class>
   <extends>QLineEdit</extends>
   <header>ctrls/LineEditEx.h</header>
  </customwidget>
  <customwidget>
   <class>CheckBoxEx</class>
   <extends>QCheckBox</extends>
   <header>ctrls/CheckBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>LabelEx</class>
   <extends>QLabel</extends>
   <header>ctrls/LabelEx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
